frontend/src/pages/NewRun.tsx

```diff
@@ -589,6 +589,7 @@ export class NewRun extends Page<NewRunProps, NewRunState> {
                     this._validate.bind(this),
                   )
                 }
+                isMaxConcurrentRunValid={true}
```

Comment:
As mentioned in another comment, this prop doesn't need to be passed in from parent component. Also the validation logic here in the KFPv1 is not implemented.